src/interfaces/prototyper.ts

```diff
@@ -0,0 +1,6 @@
+import { Feature } from './feature'
+
+export interface Prototyper {
+  generate(features: Feature[]): Promise<string>
```

Comment:
- [ ] Wouldn't `generate` supposed to return an array of file names? (one file for every feature)
- [ ] Better documenting the return meaning (JsDoc)